samples/04_gis_analysts_data_scientists/coastline_extraction-usa-landsat8_multispectral_imagery.ipynb

```diff
@@ -107,8 +107,8 @@
    "outputs": [],
```

Comment:
<p>Can you check if this works with 'Home' capitalized?</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/903/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>

Reply:
Yes it works
![image](https://user-images.githubusercontent.com/51908044/105326735-148f1000-5bf4-11eb-91db-d40ef13d2726.png)

Reply:
<p>I have updated 'Home' to 'home' for uniformity of notebooks, but both works fine.</p>